modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -27,6 +28,41 @@ const PERMISSIONS_LOCATORS = [
 	'#guest_ACTION_PERMISSIONS',
 ];
 
+test('LPD-17245: Add error message in Translation for concurrent users', async ({
+	journalEditArticlePage,
+	journalEditArticleTranslationsPage,
+	journalPage,
+	page,
+}) => {
+	await journalPage.goto();
+
+	const title = getRandomString();
+
+	await journalEditArticlePage.publishNewBasicArticle(title);
+
+	const article = page
```

Comment:
have you tested if `page.getByText(title)` works?

Reply:
As we have seen in a call, we are rendering the initial structure and then we are modifying that structure with a react component , so that's why we need a wait for here, so, we can not use waitForLoadState :(.